web/frontend/components/TheTableOfContents.vue

```diff
@@ -1,6 +1,10 @@
 <template>
 <div class="table-of-contents"
      v-bind:class="{'editable':editing}">
+     <div class="annotation-tip" v-if="this.toc.length !== 0">
```

Comment:
I think this also needs a check about whether the user is a verified professor, because it shows up for me in that case.